eos-core-depends

```diff
@@ -112,6 +112,7 @@ nautilus
 bluez-obexd
 openprinting-ppds
 openssh-server
+openvpn
```

Comment:
Okay, makes sense.  We have that in OBS, too, and it pulls in openvpn as a dependency.